### DISCUSSION TITLE
How to let visitors know when the site is down for maintenance?

### DISCUSSION THREAD

**Discussion by nogginfuel** [datenstrom/community#167] [Ask a question] [answered]
I was wondering how complex it would be to have a plugin that redirects / overrides the folder (in position 1-home or whatever) and/or entire site to a page that has a message that can be customized to let visitors know that the site is down for maintenance. Thoughts?

Example:
http://domain.com

---

This site is temporarily down for routine maintenance from **x** to **x**. It will be available again at **x**

---

or like Apple

---

Stay tuned, something great is coming...

---

**Comment by markseuffert** [comment]
Have a look at the [Draft plugin](https://github.com/annaesvensson/yellow-draft). I think it could be used it as a starting point for a maintenance plugin. You could then override individual pages with `Status: maintenance`. You could also put the entire website into maintenance mode by setting `Status: maintenance` in your `system/config/config.ini` file. 

Would that work?

**Comment by nogginfuel** [comment, marked as answer]
Thanks for the direction toward the Draft plugin. I've been taking a look at it. I'm hoping to test what I've put together within the next day or two. Hopefully it will have the desired outcome, but I suspect I might have to tweak it several times.
